app/components/forms/user-payment-info-form.js

```diff
@@ -63,6 +63,15 @@ export default class extends Component.extend(FormMixin) {
             }
           ]
         },
+        taxID: {
```

Comment:
Unnecessary Change. Revert it.

Reply:
Company field validation is also there
![Screenshot 2020-02-14 at 1 11 01 AM](https://user-images.githubusercontent.com/45634949/74471879-fa6a8f00-4ec6-11ea-9279-773999c2f3c4.png)
above code is for generating prompt for tax ID when it is left empty
here is the SS
![Screenshot 2020-02-14 at 1 16 27 AM](https://user-images.githubusercontent.com/45634949/74472275-a7dda280-4ec7-11ea-8ef7-e4a9dfc2b5d9.png)

Reply:
>above code is for generating prompt for tax ID when it is left empty

The prompt messege was already being generated, it was company field for which the messege was not being prompt . You can see the screenshot of corresponding issue.

Reply:
>Company field validation is also there

Can you share a working GIF?